cmd/antrea-agent/agent.go

```diff
@@ -638,7 +638,8 @@ func run(o *Options) error {
 	if features.DefaultFeatureGate.Enabled(features.Traceflow) {
 		packetInReasons = append(packetInReasons, uint8(openflow.PacketInReasonTF))
 	}
-	if features.DefaultFeatureGate.Enabled(features.AntreaPolicy) {
+	if features.DefaultFeatureGate.Enabled(features.AntreaPolicy) ||
+		features.DefaultFeatureGate.Enabled(features.Multicast) {
```

Comment:
I think a more proper fix is to remove the first argument of `StartPacketInHandler` and just get the reasons from the keys of `c.packetInHandlers` to avoid all the redundancy and inconsistency.

Reply:
> I think a more proper fix is to remove the first argument of StartPacketInHandler and just get the reasons from the keys of c.packetInHandlers to avoid all the redundancy and inconsistency.

I prefer this way.

Reply:
yes, it looks better. updated